spec/features/admin/merchants/index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "Visiting the Admin Merchant Index Page", type: :feature do
+  describe "User Story #24" do
+    it "displays every merchant's name within the system" do
+      # 24. Admin Merchants Index
+
+      # As an admin,
+      # When I visit the admin merchants index (/admin/merchants)
+      # Then I see the name of each merchant in the system
+
+      visit admin_merchant_index_path # I am not 100% on this routing
+
+      @merchant_1 = create(:merchant, name: "Tarzhay")
+      @merchant_2 = create(:merchant, name: "Wally World")
+      @merchant_3 = create(:merchant, name: "Ammuhzon")
+
+      within ".merchants_in_system" do  # Not sure if a `within` is necessary for this test
+        expect(page).to have_content("#{@merchant_1.name}")
```

Comment:
These are great. Maybe we could add some anchor text in the view and test that as well to make sure we have stuff showing up where we want. Great work Lito!